src/main/scala/org/broadinstitute/dsde/agora/server/GoogleCredentialHandler.scala

```diff
@@ -9,18 +9,29 @@ import com.google.api.client.json.jackson2.JacksonFactory
 import com.google.api.services.storage.StorageScopes.DEVSTORAGE_FULL_CONTROL
 
 object GoogleCredentialHandler {
-  val accessToken: String = {
+
+  val one_minute = 60
```

Comment:
variable name should be oneMinute alternately use scala.concurrent.duration 1.minute instead of the val